.github/workflows/rust.yaml

```diff
@@ -1,6 +1,16 @@
 name: Rust
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
```

Comment:
Isn't `docs` enough? It always bothers me when people make it more complicated.

Reply:
I think it is enough. Old habit from penetration testing :D
Changing it now

Reply:
Better test it though :slightly_smiling_face:

Reply:
found an even better way: - '*.md' 
😎